frontend/ui/trapper.lua

```diff
@@ -181,6 +186,16 @@ function Trapper:info(text)
     -- events won't be considered action on the yet to be displayed
     -- widget
 
+  -- If fast_refresh option, avoid UIManager refresh overhead
+  if fast_refresh and self.current_widget and self.current_widget.is_infomessage then
+    self.current_widget:free()
+    self.current_widget.text = text
+    self.current_widget:init()
+    local Screen = require("device").screen
+    self.current_widget:paintTo(Screen.bb, 0,0)
+    local d = self.current_widget[1][1].dimen
+    Screen["refreshUI"](Screen, d.x, d.y, d.w, d.h)
```

Comment:
Our style prefers the Screen.refreshUI shorthand notation. I suspiciously eye this block as looking rather complicated/low-level for where it is but I can't quite think of a better method. :-)